Solution/source/Submenus/PlayerOptions.cpp

```diff
@@ -125,6 +125,8 @@ namespace sub
 		AddToggle("Auto-Clean", loop_player_autoClean);
 		AddToggle("Super Run", loop_super_run);
 		AddToggle("Super Jump", loop_super_jump);
+		AddToggle("Walk underwater", loop_player_Walkunderwater);
+		AddToggle("Personal sun", loop_player_Psun);
```

Comment:
I'm not sure this option is necessary as an option on its own.
It doesn't really seem useful other than what you mentioned (under water in tandem with the walk-underwater option.
You can add it to the same option with the use of AddTexter.
That way it can be walk-underwater ON and walk-underwater ON WITH LIGHT (and the light will only be drawn when in water).

Just my thoughts.